src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/PropertySetAnalysis/PropertySetAnalysis.PropertySetDataFlowOperationVisitor.cs

```diff
@@ -98,49 +103,114 @@ protected override void SetAbstractValueForAssignment(IOperation target, IOperat
             public override PropertySetAbstractValue VisitObjectCreation(IObjectCreationOperation operation, object argument)
             {
                 PropertySetAbstractValue abstractValue = base.VisitObjectCreation(operation, argument);
-                if (operation.Type == this.DeserializerTypeSymbol)
+                if (operation.Type != this.TrackedTypeSymbol)
+                {
+                    return abstractValue;
+                }
+
+                ConstructorMapper constructorMapper = this.DataFlowAnalysisContext.ConstructorMapper;
```

Comment:
Just confirming: this analysis cannot be executed without providing a constructor mapper now, correct?

Reply:
Yep.  `PropertySetAnalysis.GetOrComputeHazardousUsages()` checks that constructorMapper is non-null.  Have to specify initial PropertySetAbstractValueKinds when tracked objects are instantiated.